README.md

```diff
@@ -1,55 +1,110 @@
-# Prison Visit Booking email
+# Visit someone in prison
 
 [![Build Status](https://travis-ci.org/ministryofjustice/prison-visits.png?branch=master)](https://travis-ci.org/ministryofjustice/prison-visits)
 [![Code Climate](https://codeclimate.com/github/ministryofjustice/prison-visits.png)](https://codeclimate.com/github/ministryofjustice/prison-visits)
 [![Code Coverage](https://codeclimate.com/github/ministryofjustice/prison-visits/coverage.png)](https://codeclimate.com/github/ministryofjustice/prison-visits)
 
-Also known as "PVBE", is the Alpha system for the Prison Visit Booking exemplar. 
+Also known as Prison Visits Booking, is one of the 25 [GOV.UK](https://www.gov.uk/) 'exemplar' [digital transformation services](https://www.gov.uk/transformation).
 
 It is a Ruby on Rails cookie based (no database) service which enables a user to pick 3 posible visiting times and submit the required details of a visit to the relevant prison. 
 
-The request is sent via secure email and prison staff will then manually enter the visit request into NOMIS and respond to the user.
+The request is sent via secure email to prison staff who manually process the request before returning to this app to respond to the user.
 
-## Editing prison details
+## Editing prison data
 
 Prison details are store in a single Yaml file which can be edited directly on this website by anyone with access to this repository. [Yaml files](http://en.wikipedia.org/wiki/YAML) use a two-space indentation syntax, so be careful.
 
-[Staging](https://github.com/ministryofjustice/prison-visits/blob/master/config/prison_data_staging.yml)
-[Production](https://github.com/ministryofjustice/prison-visits/blob/master/config/prison_data_production.yml)
+The [staging data](config/prison_data_staging.yml) must contain all the same parameters as [production](config/prison_data_production.yml), but not the same values.
 
-By updating this file you can:
+### Prison visibility
 
-* Add/remove a prison
-* Edit visiting slots
-* Edit non-bookable dates
-* Edit booking email address
-* Edit booking phone number
-* Edit prison address
+All known prisons should exist in the data files. If a prison is not in scope of the service, it should be disabled and can be given a specific reason.
 
-### Prison visibility
-To enable a prison, set `enabled` to `true`.
+To enable visit requests to a prison, set `enabled` to `true`.
+
+    Rochester:
+      enabled: true // this prison accepts visit requests
+    
+    Stafford:
+      enabled: false // this prison does not accept online visit request through this service
+
```

Comment:
Consider making this a [fenced code block](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) (and marking them as yaml), and using valid YAML comments (denoted with `#`).